ROLES.md

```diff
@@ -186,34 +152,37 @@ status is scoped to a part of the codebase.
 The following apply to the part of the codebase for which one would be an
 approver in an OWNERS file:
 
-- Reviewer of the codebase for at least 3 months.
+- Add yourself as a `writer` for the repositories you are responsible for
+  in [knative.yaml](../main/peribolos/knative.yaml) and/or
+  [knative-sandbox.yaml](../main/peribolos/knative-sandbox.yaml). After
+  this you should be an `approver` in the OWNERS file.
 
-- Should have reviewed or contributed to substantial PRs to the codebase.
+- Reviewer of the codebase for at least 3 months
 
-- Should be a `reviewer` in the OWNERS file.
+- Should have reviewed or contributed to substantial PRs to the codebase
```

Comment:
Question: do we want to allow people to become "Approver" without having done substantial PR reviews?  I can see both sides to this issue, and would love to hear others' opinions.

Reply:
I think the troublesome word here is "substantial". It's kind of squishy and sounds intimidating. Maybe just make this "several or important" - something that allows for wiggle room and feels less "gatekeeping".